dg/deps/config/cpd-subscriptions.json

```diff
@@ -112,7 +112,7 @@
 			"name": "ibm-cpd-datastage-operator"
 		}
 	},
-	"ibm-cpd-dods": {
+	"ibm-cpd-dods-operator": {
```

Comment:
Please revert this change as this file is machine-generated from the [documentation](https://www.ibm.com/docs/en/cloud-paks/cp-data/4.0?topic=tasks-creating-operator-subscriptions), which uses `ibm-cpd-dods` as the subscription name.